rust/ql/test/library-tests/dataflow/global/main.rs

```diff
@@ -91,6 +91,38 @@ fn data_through_method() {
     sink(b); // $ hasValueFlow=4
 }
 
+use std::ops::Add;
+
+struct MyInt {
+    value: i64,
+}
+
+impl Add for MyInt {
+    type Output = MyInt;
+
+    fn add(self, other: MyInt) -> MyInt {
+        // Ignore `other` to get value flow for `self.value`
```

Comment:
Sorry for coming up with one more nitpick comment 🙈 
```suggestion
    fn add(self, _other: MyInt) -> MyInt {
        // Ignore `_other` to get value flow for `self.value`
```